From foo@baz Mon Jan  5 10:46:00 PST 2009
Date: Mon, 05 Jan 2009 10:46:00 -0800
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: w35und: fix config build warnings

the wireless config function has changed with commit
e8975581f63870be42ff4662b293d1b0c8c21350 so fix up the wbusb driver to
work properly with that change.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/winbond/wbusb.c |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- a/drivers/staging/winbond/wbusb.c
+++ b/drivers/staging/winbond/wbusb.c
@@ -134,11 +134,12 @@ static int wbsoft_start(struct ieee80211
 	return 0;
 }
 
-static int wbsoft_config(struct ieee80211_hw *dev, struct ieee80211_conf *conf)
+static int wbsoft_config(struct ieee80211_hw *dev, u32 changed)
 {
 	struct wbsoft_priv *priv = dev->priv;
-
+	struct ieee80211_conf *conf = &dev->conf;
 	ChanInfo ch;
+
 	printk("wbsoft_config called\n");
 
 	ch.band = 1;
